src/apps/intel/intel_app.lua

```diff
@@ -44,12 +45,19 @@ function Intel82599:new (arg)
       local poolnum = firsthole(dev.vflist)-1
       local vf = dev.pf:new_vf(poolnum)
       dev.vflist[poolnum+1] = vf
-      return setmetatable({dev=vf:open(conf)}, Intel82599)
+      result = setmetatable({dev=vf:open(conf)}, Intel82599)
    else
       local dev = intel10g.new_sf(conf):open()
       if not dev then return null end
-      return setmetatable({dev=dev, zone="intel"}, Intel82599)
+      result = setmetatable({dev=dev, zone="intel"}, Intel82599)
    end
+   return result
+end
+
+function Intel82599:start ()
+   self.shmname = ("app/%s/"):format(self.appname)
```

Comment:
AFAIU the intention for the namespace under apps is that it's completely in the app's control, there's no standard division or convention that you'll be able to look for shared memory objects of a certain kind in any particular place.  To me providing for such a taxonomy is premature.